tests/helper.py

```diff
@@ -96,35 +96,39 @@ def is_isometry(tan_a, trans_a, endpoint):
     tan_vec_b = space.to_tangent(gs.random.rand(*shape) / 5, base_point)
     end_point = metric.exp(tan_vec_b, base_point)
 
-    transported = metric.parallel_transport(tan_vec_a, tan_vec_b, base_point)
+    transported = metric.parallel_transport(tan_vec_a, base_point, tan_vec_b)
+    result = is_isometry(tan_vec_a, transported, end_point)
+    results.append(gs.all(result))
+
+    transported = metric.parallel_transport(tan_vec_a, base_point, end_point=end_point)
```

Comment:
do you have a test for vectorization when called with base_point/end_point?

Reply:
Could we also add a test that checks that for the same geodesic (of initial_point, end_point and initial_tangent_vec_) both versions of the parallel transport give the same result?

Reply:
This is with multiple inputs for all args, I will try with only one `end_point`.